modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/rest/SW360Client.java

```diff
@@ -26,9 +27,11 @@
 import java.util.Collections;
 
 public abstract class SW360Client {
+    private final boolean proxyUse;
     protected RestTemplate restTemplate;
 
     public SW360Client(boolean proxyUse, String proxyHost, int proxyPort) {
+        this.proxyUse = proxyUse;
```

Comment:
To be consistent with the lines below:
```suggestion
        this.proxyUse = proxyUse && proxyHost != null;
```